models/engine/db_storage.py

```diff
@@ -74,3 +74,13 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """A method to retrieve one object"""
+        if cls and id:
```

Comment:
What happens if you receive a non string argument in get method?
To avoid type errors you can check de type of argument before using it.
try `if cls is not None and type(id) is str and id is not None:`